cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -390,9 +391,12 @@ elseif(RISCV)
   set(CPU_BASELINE "DETECT" CACHE STRING "${HELP_CPU_BASELINE}")
 
 elseif(LOONGARCH64)
+  ocv_update(CPU_LASX_TEST_FILE "${OpenCV_SOURCE_DIR}/cmake/checks/cpu_lsx.cpp")
```

Comment:
CPU_LSX_TEST_FILE?

Reply:
Thanks for your review. This is a spelling error and I have corrected it.